### ISSUE TITLE
Table captions 

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#568]
Issue group: Tables
Visibility: 1
Evaluation: Partially Supports
Success Criteria: 2.4.6: Headings and Labels
Needs design: Content

### Description
Tables caption provides a title for the table, describing the content contained it contains. 

- [ ] The tables under Messages sent all have a caption of “Total Spend”. However, it seems the caption should be something like “Total messages sent per month”?
- [x] The caption for the table on the Security keys page is “Your profile” but should be something like “Security keys”.

**Comment by adriannelee** [comment]
Few PRs for this and other a11y issues. @amazingphilippe will link the PRs and move them into Review

**Comment by amazingphilippe** [comment]
Fix for the security key table is in this PR: https://github.com/cds-snc/notification-admin/pull/1734

Once merged, move back to backlog so we can tackle the other table caption described above
